src/components/common/TmConnectedNetwork.vue

```diff
@@ -68,19 +68,59 @@
         Connecting…
       </div>
     </div>
+    <hr v-if="currentNetwork.powered" class="powered-hr" />
+    <div v-if="currentNetwork.powered" class="powered-div">
+      <span class="powered-by">Powered by&nbsp;</span>
```

Comment:
is the `&nbsp;` necessary?

Reply:
no, it was just a leftover from first version. Good catch